rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -117,6 +117,7 @@ namespace rsx
 				return sizeof(u16) * size;
 			case 3:
 				return sizeof(u16) * 4;
+			default: break;
```

Comment:
ASSUME(0)

Reply:
Nothing is stopping vertex size from being invalid, throwing is correct here.

Reply:
What should I throw? Are there any special voodoo exception types in rpcs3?

Reply:
Just break is fine, the throw is already present below.